configure

```diff
@@ -4575,7 +4585,7 @@ then :
   { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for $CXX option to enable C++11 features" >&5
 printf %s "checking for $CXX option to enable C++11 features... " >&6; }
 
-ac_cv_prog_cxx_11=no
+ac_cv_prog_cxx_cxx11=no
```

Comment:
Seems the 2.71 is patched as well, this comes from:
 https://sources.debian.org/src/autoconf/2.71-3/debian/patches/fix-cxx-stdcxx-test.diff/